### PR TITLE
Remove feature flags for conditional update and conditional create.

### PR DESCRIPTION
**Pull request by Ivanidzo4ka**
## Description
Remove feature flags for conditional update/create.

## Related issues
Addresses [#813]

## Testing
Through already existing unit tests.

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Shared.Web/appsettings.json [hunk 1]
```diff
@@ -82,6 +80,12 @@
             "MaxWaitTimeInSeconds": 5
         }
     },
+    "SqlServer": {
```
Comment (Ivanidzo4ka) [inline src/Microsoft.Health.Fhir.Shared.Web/appsettings.json:83]: revert me

File: src/Microsoft.Health.Fhir.Shared.Api/Modules/FeatureFlags/ConditionalCreates/ConditionalCreatePostConfigureOptions.cs [hunk 2]
```diff
@@ -17,40 +16,32 @@ namespace Microsoft.Health.Fhir.Api.Modules.FeatureFlags.ConditionalCreates
...
+            // Turns on conditional creates, even when not configured in the DefaultCapabilities.json
+            // If this is not enabled in the capability statement
+            // ConditionalCreateResourceRequest will fail with a MethodNotAllowedException
+            _configuredConformanceProvider
+                .ConfigureOptionalCapabilities(x =>
+                {
+                    foreach (ListedResourceComponent r in x.Rest.Server().Resource)
                     {
-                        foreach (ListedResourceComponent r in x.Rest.Server().Resource)
+                        if (r.Interaction.Any(y => string.Equals(y.Code, TypeRestfulInteraction.Create, StringComparison.Ordinal)))
```
Comment (brendankowitz) [inline src/Microsoft.Health.Fhir.Shared.Api/Modules/FeatureFlags/ConditionalCreates/ConditionalCreatePostConfigureOptions.cs:39]: I'd probably just incorporate this into CapabilityStatementBuilder now

Comment (Ivanidzo4ka) [reply src/Microsoft.Health.Fhir.Shared.Api/Modules/FeatureFlags/ConditionalCreates/ConditionalCreatePostConfigureOptions.cs]: I was looking where to put it, thanks for pointing out!

**Review by brendankowitz — APPROVED** [review]
:shipit:

**Pull request merged**